test/utils/SchemaTester.js

```diff
@@ -0,0 +1,108 @@
+const testSchema = (Model) => ({
+    fieldRequired: (fieldName, customObject) => {
+        test(`'${fieldName}' is required`, async () => {
+            const emptyObject = new Model(customObject || {});
+            try {
+                await emptyObject.validate();
+            } catch (err) {
+                expect(err.errors[fieldName]).toBeDefined();
+                expect(err.errors[fieldName]).toHaveProperty("kind", "required");
+                expect(err.errors[fieldName]).toHaveProperty("message", `Path \`${fieldName}\` is required.`);
+            }
+        });
+    },
+
+    minLength: (fieldName, minimum) => {
+        describe(`${fieldName} must have a minimum of ${minimum} characters`, () => {
+            test("check for error", async () => {
```

Comment:
```suggestion
            test("less than minimum bound should throw error", async () => {
```

Clearer name, I think